cmd/snap-update-ns/change.go

```diff
@@ -21,6 +21,7 @@ package main
 
 import (
 	"fmt"
+	"os"
 	"path"
```

Comment:
why is this using `path` and not `path/filepath`?

Reply:
Hmm, not sure, is there a difference?